cdap-api/src/main/java/co/cask/cdap/api/worker/Worker.java

```diff
@@ -53,7 +53,8 @@
   void destroy();
 
   /**
-   * Request to stop the running worker.
+   * Stop the {@code Worker}. This method will be invoked whenever the worker is externally stopped by CDAP.
+   *
    * This method will be invoked from a different thread than the one calling the {@link #run()} method.
```

Comment:
I'm wondering about this link here: `{@link #run()}` : it is resolving to `Runnable.run()` or http://docs.oracle.com/javase/7/docs/api/java/lang/Runnable.html?is-external=true#run().

That could be correct, but I'm not sure. Could you find out (may need to ask @chtyim)?